code/processes/gateway.q

```diff
@@ -74,6 +74,7 @@
 
 \d .gw
 
+formatresponse:@[value;`.gw.formatresponse;{{[status;sync;result] :$[not[status]and sync=`sync;'result;result];}}]
```

Comment:
maybe add a comment to this line to explain the default behaviour?

Reply:
Don't need the colon or semi colon in a single line lambda.

Reply:
Yes, comment to define default would be good.
Explain what the parameters are e.g. status is 1b for success and 0b for error etc.